docs/data/material/getting-started/support/support.md

```diff
@@ -66,12 +66,13 @@ Find details on the [supported versions](/versions/#supported-versions).
 MUI X comes in [three plans](https://mui.com/pricing/).
 The support available on the Community plan is mostly made possible thanks to people like you: the community. It's described in more detail in the above section. The maintainers of MUI focuses on solving the root issues, not direct support.
```

Comment:
```suggestion
The support available on the Community plan is made possible thanks to people like you: the community. 
It's described in more detail in the section above. 
MUI X maintainers focus on solving root issues rather than offering direct support to the community at large.
If you need support from us, you'll need to purchase one of the commercial licenses described below.
```

Reply:
"If you need support from us, you'll need to purchase one of the commercial licenses described below."

This is something we need to clarify, that by buying the commercial licenses, you don't get support on the Core components.

```suggestion
The support available on the Community plan is made possible thanks to people like you: the community.
It's described in more detail in the section above. 
MUI X maintainers focus on solving root issues rather than offering direct support to the community at large.
```